ui/pages/settings/developer-options-tab/developer-options-tab.tsx

```diff
@@ -230,6 +273,7 @@ const DeveloperOptionsTab = () => {
         {renderAnnouncementReset()}
         {renderOnboardingReset()}
         {renderServiceWorkerKeepAliveToggle()}
+        {renderNetworkMenuRedesign()}
```

Comment:
Will users be able to enable these settings?  Worried they'll see things in a broken half implemented state